### PR TITLE
Added support of Numeral for ka_GE

### PR DESCRIPTION
**Pull request by desmart18**
Summary:
* `ka_GE` locale
* full support for Numeral

### CODE REVIEW

**Comment by stopachka** [comment]
Gabarjoba David! 

Happy to see a Georgian student into Haskell!

File: Duckling/Numeral/KA/Corpus.hs [hunk 1]
```diff
@@ -35,9 +37,378 @@ allExamples = concat
   , examples (NumeralValue 2)
              [ "2"
              , "ორი"
+             , "წყვილი"
+             , "წყვილები"
              ]
   , examples (NumeralValue 3)
              [ "3"
              , "სამი"
+             , "ცოტა"
```
Comment (stopachka) [inline Duckling/Numeral/KA/Corpus.hs]: I don't know if ცოტა, რამდენიმე etc is commonly used as three in Georgian. Is that true?

From what I understand these mean "a few" etc -- cc @patapizza how do we deal with that in other languages? -- "a little bit" "a few" etc

Comment (patapizza) [reply Duckling/Numeral/KA/Corpus.hs]: @stopachka Ha got it, in English we also do "a few" = 3 :)

**Comment by patapizza** [comment]
გმადლობთ @desmart18!

**Pull request closed without merging**